pkgs/os-specific/linux/usbguard/default.nix

```diff
@@ -23,10 +26,9 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [
-    libxslt
-    asciidoctor
```

Comment:
Were `asciidoctor` and `pandoc` just not being used?

Reply:
@srhb 

- asciidoctor: usbguard 0.7.3 moved from asciidoctor to asciidoc, but the build fails with asciidoc present
- pandoc: not used except in `src/Tests/Source/CheckScripts/spell-check.sh` which doesn't appear to be run

Thanks for the review.